addons/payment_adyen/controllers/__init__.py

```diff
@@ -1,3 +1,3 @@
-# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
And why not
``` python3
# -*- coding: utf-8 -*-
# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Reply:
I'm ok to reintroduce them in practically untouched files, so that the Github's History tab is cleaner, but do the others have any actual role in Odoo's python 3 ?